examples/plotting/aspect.py

```diff
@@ -1,3 +1,11 @@
+''' This example demonstrates how a circle with a data-space radius appears when plotted with different aspect scales specified.
```

Comment:
line length (apologies I have not been checking this as consistently as I should) 
```suggestion
''' This example demonstrates how a circle with a data-space radius appears
when plotted with different aspect scales specified.
```

Reply:
Done.